modules/afsql/afsql.c

```diff
@@ -1478,12 +1479,17 @@ afsql_dd_new(GlobalConfig *cfg)
 gint
 afsql_dd_lookup_flag(const gchar *flag)
 {
-  if (strcmp(flag, "explicit-commits") == 0 || strcmp(flag, "explicit_commits") == 0)
-    return AFSQL_DDF_EXPLICIT_COMMITS;
-  else if (strcmp(flag, "dont-create-tables") == 0 || strcmp(flag, "dont_create_tables") == 0)
-    return AFSQL_DDF_DONT_CREATE_TABLES;
+  gint retval = 0;
+  const gchar *flag_copy = __normalize_key(flag);
+
+  if (strcmp(flag_copy, "explicit_commits") == 0)
+    retval = AFSQL_DDF_EXPLICIT_COMMITS;
+  else if (strcmp(flag_copy, "dont_create_tables") == 0)
+    retval =  AFSQL_DDF_DONT_CREATE_TABLES;
```

Comment:
There is another one in `log_writer_options_lookup_flag`: `no_multi-line`

Reply:
Oh I see the same problem exists in that log_writer_options_lookup_flag too. Fine for me, I will fix that too.

Reply:
Thanks.